server/webapp/WEB-INF/spring-cruise-remoting-servlet.xml

```diff
@@ -82,6 +88,17 @@
         </property>
     </bean>
 
+    <bean name="recomputeMaterialPipelineMappingsThread" class="org.springframework.scheduling.timer.TimerFactoryBean">
+        <property name="scheduledTimerTasks">
+            <list>
+                <bean class="org.springframework.scheduling.timer.ScheduledTimerTask"
+                      p:delay="10000"
```

Comment:
Is there any reason to make this a property? Part of cruise.properties or whatever that file is.